src/simulate.jl

```diff
@@ -102,10 +102,10 @@ end
 Returns the appropriate simulation step function for the given `dt1` based on a deterministic schedule.
 This function determines whether vehicles should drive or plug based on time of day.
 """
-function get_simustep_deterministic(dt1::DateTime)
+function get_simustep_deterministic(dt1::DateTime, drive_starts_time, park_starts_time)
     date_part = Dates.Date(dt1)
-    dt_9am = DateTime(date_part, Dates.Time(9, 0, 0))
-    dt_5pm = DateTime(date_part, Dates.Time(17, 0, 0))
+    dt_9am = DateTime(date_part, drive_starts_time)
```

Comment:
A bit confusing to call these dt_9am and dt_5pm. Maybe `dt_drive_start` and `dt_park_start`?